### PR TITLE
Replaced RDA with bibframe responsibilityStatement

### LINKED ISSUE #233: Replace rdau:P60339 with bf:responsibilityStatement

### PR DESCRIPTION
**Pull request by ChristophEwertowski**
Replaced http://rdaregistry.info/Elements/u/P60339 with
http://id.loc.gov/ontologies/bibframe/responsibilityStatement.

Resolves #233 after conflicts are going to be solved.

### CODE REVIEW

**Comment by acka47** [comment]
+1

File: src/main/resources/labels/labels.json [hunk 1]
```diff
@@ -6442,9 +6442,9 @@
         "multilangLabel": { }
     },
     {
-        "uri": "http://rdaregistry.info/Elements/u/P60339",
+        "uri": "http://id.loc.gov/ontologies/bibframe/responsibilityStatement",
         "label": "Verantwortlich",
-        "name": "statementOfResponsibility",
+        "name": "responsibilityStatement",
         "referenceType": "String",
```
Comment (dr0i) [inline src/main/resources/labels/labels.json]: This entry is a doublete. Going to remove it.

**Comment by dr0i** [comment]
@fsteeg wrongly already merged into master. But my guess is that there is nothing to update on UI's side, yes?

**Comment by fsteeg** [comment]
Assigned and reopened issue https://github.com/hbz/lobid-resources/issues/233#issuecomment-286069544

**Pull request merged**